.github/workflows/gradle.yml

```diff
@@ -37,5 +35,43 @@ jobs:
       run: sudo apt-get install omlib-modelica-3.0
     - name: Grant execute permission for gradlew
       run: chmod +x gradlew
-    - name: Test with Gradle
-      run: ./gradlew test -i
+    - name: Build with Gradle
+      run: ./gradlew build -i
+  release:
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check out ${{ github.ref }}
+      uses: actions/checkout@v2
+    - name: Set up JDK 11
+      uses: actions/setup-java@v2
+      with:
+        java-version: '11'
+        distribution: 'adopt'
+    - name: Grant execute permission for gradlew
+      run: chmod +x gradlew
+    - name: Build Jars with Gradle
+      run: ./gradlew shadowClientJar shadowServerJar
+    - name: Add version from build.gradle to output
+      id: version
+      run: echo "::set-output name=version::$(./gradlew getVersion -q)"
```

Comment:
I like the idea of getting the version from Gradle, but you didn't have to write a separate gradle task for this. This line of bash code would also do the trick:

```
./gradlew properties | grep version | sed "s/version: //"
```

Grep selects the line containing the version, the sed command replaces the string "version: " that is in front of the actual version number with the empty string.